config/cesm/machines/config_pio.xml

```diff
@@ -44,12 +44,6 @@
   <entry id="PIO_TYPENAME">
     <values>
       <value>pnetcdf</value>
-      <value mach="userdefined">netcdf</value>
```

Comment:
These defaults are no longer required, they will be set by the lack of a parallel option.